ui/app/AppLayouts/Browser/popups/BrowserWalletMenu.qml

```diff
@@ -7,25 +7,32 @@ import StatusQ
 import StatusQ.Controls
 import StatusQ.Core
 import StatusQ.Core.Theme
-import StatusQ.Core.Utils as SQUtils
 
 import shared.controls
 import shared.views
 import shared.stores as SharedStores
 import utils
 
-import AppLayouts.Browser.stores as BrowserStores
-
 // TODO: replace with StatusMenu
 Dialog {
     id: root
 
-    required property BrowserStores.BrowserWalletStore browserWalletStore
-    required property BrowserStores.BrowserActivityStore browserActivityStore
+    required property var accounts
+    required property var overview
+    required property var activityStore
+    required property var transactionActivityStatus
+    required property SharedStores.CurrenciesStore currencyStore
+    required property SharedStores.NetworksStore networksStore
 
     signal sendTriggered(string address)
     signal reload()
     signal accountChanged(string newAddress)
+    signal accountSwitchRequested(string address)
+    signal filterAddressesChangeRequested(string addressesJson)
+    signal transactionFilterUpdateRequested()
+    signal collectiblesModelUpdateRequested()
+    signal recipientsModelUpdateRequested()
+    signal connectedAccountDeleted()
```

Comment:
As a naming suggestion to keep the pattern consistent, you could use `deleteConnectedAccountRequested()`. But feel free to decide what fits best!